### DISCUSSION TITLE
Release 25.01-rc1

### DISCUSSION THREAD

**Discussion by svpcom** [svpcom/wfb-ng#394] [General]
## Release notes:
- Added distributed mode. Now you can use any amount of hosts with any amount of wifi adapters as single GS or Drone instance.
- Added support for OpenWrt. Now you can use OpenWrt 24.10-based router as RX-only (most of routers) or RX/TX (only ath9k SoC based)  node in distributed mode.
- Added JSON API for link state telemetry integration with side projects.
- Radxa Zero 3W/3E and are Orange PI5 supported now in wfb-ng-osd (via libdrm hw overlay on top of gstreamer window)

<hr /><em>This discussion was created from the release <a href='https://github.com/svpcom/wfb-ng/releases/tag/wfb-ng-25.01-rc1'>Release 25.01-rc1</a>.</em>

**Comment by lida2003** [comment]
It's nice to see [wfb-ng_25.01-rpi3.img.xz](https://github.com/svpcom/wfb-ng/releases/download/wfb-ng-25.01-rc1/wfb-ng_25.01-rpi3.img.xz).

Do you have any test data or feedback result about FPS/latency, such as 1080P@120FPS 35ms latency?

**Reply by svpcom** [reply]
No. I don't have 120 or 240 Hz monitor to do measurements. My old and slow 60Hz monitor will show large latency anyway

**Reply by lida2003** [reply]
Hope to see such data. It's nice to have 3588(6Tops) airunit support.
